tools/dbus_exporter.py

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env python3
 # SPDX-License-Identifier: LGPL-2.1-or-later
+import os
 from argparse import ArgumentParser
 from pathlib import Path
 from subprocess import PIPE, run
 
 
 def extract_interfaces_xml(output_dir, executable):
+    # If proc is not mounted, set LD_LIBRARY_PATH so that shared/core libs can be found
```

Comment:
Please explicitly mention that glibc looks at /proc/self/exe when resolving RPATH